docs/interfaces.md

```diff
@@ -107,6 +107,17 @@ Auto-Connect: no
 
 ## Supported Interfaces - Advanced
 
+### browser
+
+Can access files and IPC needed by modern browsers. This interface is
+intended to be used when using an embedded Chromium Content API or using the
+sandboxes in major browsers from vendors like Google and Mozilla.
+
+Usage: reserved
+Auto-Connect: yes
+Attributes:
+- allow-browser-sandbox: true|false (defaults to ``false``)
```

Comment:
Re `sandbox: true`, I initially did that, but then I though people might get confused and think it referred to the snappy sandbox. Happy to change it if you think that isn't an issue.